dot/rpc/modules/api.go

```diff
@@ -33,6 +33,7 @@ type StorageAPI interface {
 }
 
 //go:generate mockery --name BlockAPI --structname BlockAPI --case underscore --keeptree
+//go:generate mockgen -destination=mock_block_api_test.go -package=$GOPACKAGE . BlockAPI
```

Comment:
Maybe have this newer mockgen command using the format described in https://engineering-handbook.onrender.com/development/tech-stack/go#mock-generation ?

That way there would be less diffs in the future once we migrate fully to having all mockgen mocks in `mocks_test.go`.